python/hsfs/feature_group.py

```diff
@@ -154,9 +160,26 @@ def insert(self, features, overwrite=False, storage=None, write_options={}):
             write_options,
         )
 
+        if self.statistics_config.enabled:
+            self._statistics_engine.compute_statistics(self, self.read(storage))
```

Comment:
nitpick - we can call the compute_statistics method here.

Reply:
done